.changeset/fresh-plants-shave.md

```diff
@@ -0,0 +1,7 @@
+---
+'@siafoundation/renterd-types': minor
+'@siafoundation/types': minor
+'@siafoundation/renterd': minor
```

Comment:
No need for renterd app in here since its not directly a feature change, the dependency on the types will bump the app automatically.

Reply:
Got it!